templates/miq-template.yaml

```diff
@@ -88,7 +88,7 @@ objects:
             persistentVolumeClaim:
               claimName: ${NAME}
         containers:
-        - image: fbladilo/miq-app:latest
+        - image: fbladilo/miq-app:${APPLICATION_IMG_TAG}
```

Comment:
@simaishi I removed the image key altogether for the 3 pod templates, we rely on the deployment config Image params to call the correct image from the ImageStream here : 

https://github.com/fbladilo/manageiq-pods/blob/master/templates/miq-template.yaml#L160

I tested without the container spec image key defined and worked as expected, the DC populated the pod template image key with the requested image/tag from IS : 
```bash
[root@franco-ocp-eval-master templates (template_add_version_tag_param)]# oc describe pods manageiq-2-5725m | grep Image
    Image:		docker.io/fbladilo/miq-app@sha256:395eb0dd8306b1d3c30ab0de884b7a4091746ab8c4641be4156b839f404065a9
    Image ID:		docker://sha256:c32c91aec7a6d77cb3aca608d987e4e42deb51dd4b1a61481df3a515abf8ceb1
```